calibPipe.py

```diff
@@ -0,0 +1,67 @@
+
+import MySQLdb
```

Comment:
Note this is a very standalone prototype with no dependency on stack, it is not even in github.com:lsst, but in github.com:lsst-dm. I will switch once we make this part of stack. Deal?